### ISSUE TITLE
Add support for WPS

### ISSUE THREAD

**Issue by cehbrecht** [geopython/GeoHealthCheck#203]
I can work on WPS support. We need this for our project.

**Comment by justb4** [comment]
@cehbrecht excellent! Guess you mean adding Probe and Check Plugins for health-checking WPS endpoints?

**Comment by cehbrecht** [comment]
@justb4
> Guess you mean adding Probe and Check Plugins for health-checking WPS endpoints?

 ... If you say so, just thought GeoHealthCheck is the right tool to check the availability of WPS services.

**Comment by justb4** [comment]
@cehbrecht yes it is:-), you can find Probe Plugins for most OGC services here:
https://github.com/geopython/GeoHealthCheck/tree/master/GeoHealthCheck/plugins/probe

There is already a WPS Capabilities Probe:
https://github.com/geopython/GeoHealthCheck/blob/master/GeoHealthCheck/plugins/probe/owsgetcaps.py#L159, but you may want to add additional Probes, or maybe a drilldown/crawler. Challenge may be long-running WPS Processes (though with an upcoming PR #175 we will have per-Probe scheduling) and async, but I'm no expert in WPS.
